src/StorageSync/StorageSync/ServerEndpoint/SetServerEndpointCommand.cs

```diff
@@ -0,0 +1,175 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.StorageSync.Common;
+using Microsoft.Azure.Commands.StorageSync.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.StorageSync.Common.Extensions;
+using Microsoft.Azure.Commands.StorageSync.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.StorageSync;
+using Microsoft.Azure.Management.StorageSync.Models;
+using System.Management.Automation;
+using StorageSyncModels = Microsoft.Azure.Management.StorageSync.Models;
+
+namespace Microsoft.Azure.Commands.StorageSync.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Set, StorageSyncNouns.NounAzureRmStorageSyncServerEndpoint, DefaultParameterSetName = StorageSyncParameterSets.ObjectParameterSet), OutputType(typeof(PSServerEndpoint))]
```

Comment:
must support shouldprocess

Reply:
duplicate

---
In reply to: [255704782](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255704782) [](ancestors = 255704782)